Source/SwiftLintFramework/ViolationSeverity.swift

```diff
@@ -28,8 +28,12 @@ public enum ViolationSeverity: Int, Printable, Comparable {
         }
     }
 
+    public var isError: Bool {
+        return self > Medium
+    }
+
     public var xcodeSeverityDescription: String {
-        return self <= Medium ? "warning" : "error"
+        return self.isError ? "error" : "warning"
```

Comment:
please omit the `self` keyword here. (it'd be nice to add a rule for this at some point :smile:)